test/express-tests.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
```

Comment:
I'd rather specify a global timeout in Mocha than a line in every test. Leave it for now, but once a different PR is merged in, I think there'll be a good way to fix this.